api/stream/design.go

```diff
@@ -662,6 +662,10 @@ var ListSourcesRequest = Type("ListSourcesRequest", func() {
 var UpdateSourceRequest = Type("UpdateSourceRequest", func() {
 	SourceKeyRequest()
 	SourceFieldsRW()
+	Attribute("changeDescription", String, func() {
+		Description("Description of the modification, description of the version.")
+		Example("Renamed.")
+	})
 })
```

Comment:
Added mission `changeDescription` API field, it describes the new version of the entity.